include/micm/solver/rosenbrock.inl

```diff
@@ -91,7 +97,7 @@ namespace micm
           double stage_combinations = ((stage + 1) - 1) * ((stage + 1) - 2) / 2;
           if (stage == 0)
           {
-            K[stage].Copy(initial_forcing);
+            K[stage].Swap(initial_forcing);
```

Comment:
If the time step is rejected twice, I think the `K[0]` vector is not initialized to `initial_forcing` any more?

Reply:
I think you're right. I didn't think far enough ahead, I suppose